python/sglang/srt/managers/io_struct.py

```diff
@@ -542,6 +542,11 @@ class UpdateWeightsFromDistributedReqInput:
     shape: List[int]
 
 
+@dataclass
+class BatchUpdateWeightsFromDistributedReqInput:
+    parameters: List[Dict[str, Union[str, List[int]]]]
```

Comment:
is it possible to have
```suggestion
    parameters: List[UpdateWeightsFromDistributedReqInput]
```